### DISCUSSION TITLE
Is Voyager now end of life?

### DISCUSSION THREAD

**Discussion by iugo-robert** [thedevdojo/voyager#5360] [General]
Hi,

We've noticed that there hasn't been any recent releases and the last release of Voyager was 1.4.2 on 2 July 2020.

Is it likely that Voyager will still get active support and maintenance into the future?

Thanks,
Rob

**Comment by emptynick** [comment]
Nope, we are still merging PRs and fixes.
Also, we are putting most of our time into Voyager 2 at the moment.
